image/docs/containers-signature.5.md

```diff
@@ -67,10 +67,11 @@ When verifying a cryptographic signature in the OpenPGP format,
 the consumer MUST verify at least the following aspects of the signature
 (like the `go.podman.io/image/v5/signature` package does):
 
-- The blob MUST be a “Signed Message” as defined RFC 4880 section 11.3.
+- The blob MUST be a “Signed Message” as defined in RFC 4880 section 11.3.
   (e.g. it MUST NOT be an unsigned “Literal Message”,
   a “Cleartext Signature” as defined in RFC 4880 section 7,
   or any other non-signature format).
+- The blob MUST contain one single "Signature Packet" as defined in RFC 4880 section 5.2.
```

Comment:
‘must not contain multiple ”Signature Packet”s’, to be unambiguous?

As a non-native speaker, I don’t read “must contain one” to prohibit “must not contain two”; the “single” sort of does imply uniqueness, but that kind of grammar is not taught in introductory English courses.

Reply:
@mtrmac I just reworded it